README.md

```diff
@@ -1,2 +1,49 @@
+[![Build Status](https://travis-ci.org/ucsc-cgp/bouncer.svg?branch=master)](https://travis-ci.org/ucsc-cgp/bouncer)
+
 # bouncer
 The whitelist checker for authentication with CGP HCA Data Store
```

Comment:
How about this as a title:
Simple email whitelist check backed by the AWS Secrets Manager

Reply:
I like it!